Assets/Text/cqui_InGameText_ja.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Indentation seems to be wrong here. I think when I copied the original file, I did not replace the tabs by 2 spaces.

Reply:
Yes, the indentation is supposed to be tow spaces, not tab with a size of two. In all the files.